core/bentley/src/UtilityTypes.ts

```diff
@@ -102,3 +102,16 @@ export type PromiseReturnType<T extends AsyncFunction> = T extends (...args: any
  * @beta
  */
 export type ExtractLiterals<T, U extends T> = Extract<T, U>;
+
+/** A runtime property omitter, removing properties in a given list from a given object.
```

Comment:
Misleading doc. It does not remove properties from a given object, it creates and returns a shallow copy of that object with those properties removed.

Reply:
updated